go.mod

```diff
@@ -8,12 +8,11 @@ require (
 	github.com/gookit/color v1.5.0
 	github.com/mholt/archiver v3.1.1+incompatible
 	github.com/schollz/progressbar/v3 v3.8.5
-	github.com/urfave/cli v1.22.5
```

Comment:
I think I would prefer to use v2 instead of the v3 alpha. Looks like v3 has been in alpha for a few years now.